ci/check-documentation.py

```diff
@@ -114,6 +116,21 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
+        # check <omz_dir> reference validity
+
+        for link in doc_page.code_spans():
```

Comment:
```suggestion
        for code_span in doc_page.code_spans():
```

This variable can contain values that are not links.